com.unity.multiplayer.mlapi/Editor/ProfilerModuleOverride/NetworkProfilerOverride.cs

```diff
@@ -0,0 +1,249 @@
+using System;
+using System.Collections.Generic;
+using MLAPI.Profiling;
+using Unity.Profiling;
+using UnityEditor;
+using UnityEngine;
+
+namespace ProfilerModuleOverride
+{
+    [InitializeOnLoad]
+    public class NetworkProfilerOverride
+    {
+#if UNITY_2020_2_OR_NEWER
+        const string RPCModuleName = "MLAPI RPCs";
+        const string OperationModuleName = "MLAPI Operations";
+        const string MessageModuleName = "MLAPI Messages";
+
+        [System.Serializable]
+        public class MLAPIProfilerCounter
+        {
+            public string m_Name;
+            public string m_Category;
+        }
+
+        [System.Serializable]
+        public class MLAPIProfilerModuleData
+        {
+            public MLAPIProfilerModuleData()
```

Comment:
nit, this could be done with field initializers to improve code readability

Reply:
Done